go.mod

```diff
@@ -2,15 +2,13 @@ module github.com/nats-io/nats-account-server
 
 require (
 	github.com/fsnotify/fsnotify v1.4.7
-	github.com/golang/protobuf v1.3.1 // indirect
 	github.com/julienschmidt/httprouter v1.2.0
 	github.com/mitchellh/go-homedir v1.0.0
-	github.com/nats-io/gnatsd v1.4.1 // indirect
-	github.com/nats-io/go-nats v1.7.2
-	github.com/nats-io/jwt v0.2.4
-	github.com/nats-io/nats-server v0.0.0-20190506224138-6584a9a8283f
+	github.com/nats-io/jwt v0.2.6
+	github.com/nats-io/nats-server/v2 v2.0.0
+	github.com/nats-io/nats.go v1.8.1
 	github.com/nats-io/nkeys v0.0.2
-	github.com/nats-io/nsc v0.0.0-20181222012937-6aff930ee7ef
+	github.com/nats-io/nsc v0.0.0-20190605004402-38261a2a753e
```

Comment:
We should be tagging `nsc` repo with semver notation (e.g. `v0.3.4` instead of `0.3.4`) so that `go.mod` picks up the proper tag instead of the `v0.0.0` pseudo version: https://github.com/nats-io/nsc/tags